go/vumitools/account/models.py

```diff
@@ -26,10 +26,23 @@ class UserAppPermission(Model):
     application = Unicode(max_length=255)
 
 
+def flag_method(name):
+    def fget(self):
+        return name in self.flags
+
+    def fset(self, value):
+        if value:
+            self.flags.append(name)
+        else:
+            del self.flags[list(self.flags).index(name)]
```

Comment:
`ListProxy` doesn't have a `.remove`, thus the need for this weird way of removing the flag. This will go away when we add `SetOf`, which I'm assuming will happen before this PR lands.